src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -99,7 +99,10 @@ def __call__(self, organization, event):
                                 author_email = user.email
                                 gh_username_cache[gh_username] = author_email
                                 if commit_author is not None:
-                                    commit_author.update(email=author_email)
+                                    commit_author.update(
+                                        email=author_email,
+                                        external_id=gh_username,
```

Comment:
> what if your phabricator username was github:matt :trollface:

This wouldn't matter since we'd be storing it as `phabricator:github:matt` instead. :)